content/pt-br/docs/concepts/services-networking/ingress.md

```diff
@@ -0,0 +1,528 @@
+---
+title: Ingress
+content_type: concept
+description: >-
+  Disponibilize seu serviço de rede HTTP ou HTTPS usando um mecanismo de configuração com reconhecimento de protocolo, que entende conceitos da Web como URIs, nomes de host, caminhos e muito mais. 
+  O conceito Ingress permite mapear o tráfego para diferentes backends com base nas regras definidas por meio da API do Kubernetes.
+weight: 30
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.19" state="stable" >}}
+{{< glossary_definition term_id="ingress" length="all" >}}
+
+
+<!-- body -->
+
+## Terminologia
+
+Para fins de clareza, este guia define os seguintes termos:
+
+* Nó: Uma máquina de trabalho no Kubernetes, parte de um cluster.
+* Cluster: Um conjunto de nós que executam aplicações em contêiner gerenciado pelo Kubernetes. Para este exemplo, e nas instalações mais comuns do Kubernetes, os nós no cluster não fazem parte da Internet pública.
+* Roteador de borda: Um roteador que impõe a política de firewall para o seu cluster. Isso pode ser um gateway gerenciado por um provedor de nuvem ou um  hardware físico.
+* Rede do cluster: Um conjunto de links, lógicos ou físicos, que facilitam a comunicação dentro de um cluster de acordo com o [modelo de rede](/pt-br/docs/concepts/cluster-administration/networking/) do Kubernetes.
+* Serviço: Um {{< glossary_tooltip text="serviço" term_id="service" >}} Kubernetes que identifica um conjunto de Pods usando seletores de {{< glossary_tooltip text="label" term_id="label" >}}. Salvo indicação em contrário, assume-se que os Serviços tenham IPs virtuais apenas roteáveis dentro da rede de cluster.
```

Comment:
hum, this "Um service Kubernetes", maybe "Um 'objeto Service do Kubernetes'"

Reply:
+1 for Katz suggestion